mozilla-mobile/android-components/components/ui/tabcounter/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Message announced to the user when tab tray is selected with 1 tab -->
+    <string name="mozac_tab_counter_open_tab_tray_single">1 open tab. Tap to switch tabs.</string>
+    <!-- Message announced to the user when tab tray is selected with 0 or 2+ tabs -->
+    <string name="mozac_tab_counter_open_tab_tray_plural">%1$s open tabs. Tap to switch tabs.</string>
```

Comment:
This assumes that all languages use the same plural rules (as in 0 and >2 are plurals, and 1 is singular: [see this comment/ceonversation](https://github.com/mozilla-l10n/android-l10n/pull/11/files#r271636736)).
Since this is not the case - and we still unfortunately don't support plurals on Android - I would suggest to work around this by special casing 0 and 1, and using a string with parameter for the rest.
CC @eliserichards for an update on the strings :)

Reply:
These are strings taken directly from Fenix, so I will have to check with Betsy for any copy changes :) thanks for letting me know!

Reply:
Hi @eliserichards! Yes, this is correct, strings come from Fenix and arrive in quarantine here so I can review them before exposing them to localizers.
This is the usual process, so maybe I'm misunderstanding your comment? :)  Let me know! cheers

Reply:
Update: just realized these strings live in a-c. Now I understand what you were saying :) Sorry for the confusion. I believe they should then also be udpated on Fenix side, maybe I didn't catch them. I'll confirm tomorrow about this! thanks

Reply:
@eliserichards : Quick update - since we still don't support plurals correctly, an easy way to work around this would be to just update the comment for "mozac_tab_counter_open_tab_tray_plural" to something like "Message announced to the user when tab tray is selected with multiple tabs"
That will help localizers work around our lack of plural support

Reply:
The plan is to update the comment for each of these plurals. Is there a chance that the translations will be available for our beta next week @Delphine